.github/workflows/system-tests-backend.yml

```diff
@@ -24,6 +24,10 @@ jobs:
         agent-auto-init: [ "true" ]
         include:
           - sample: "sentry-samples-spring-boot"
+          - sample: "sentry-samples-spring-boot-opentelemetry-noagent"
+          - sample: "sentry-samples-spring-boot-opentelemetry"
```

Comment:
Do we also want to have a test with `agent-auto-init: "false"` like for spring boot 3?

Reply:
Yeah